lib/bambora/client.rb

```diff
@@ -1,47 +1,101 @@
 # frozen_string_literal: true
 
-require 'base64'
-require 'faraday'
+require 'bambora/rest_client'
+require 'bambora/headers'
+require 'bambora/json_client'
+require 'bambora/v1/batch_payment_report_resource'
+require 'bambora/v1/batch_payment_resource'
+require 'bambora/v1/profile_resource'
+require 'bambora/v1/payment_resource'
+require 'bambora/client/version'
 
 module Bambora
```

Comment:
`Bambora::Client` is more of a convenience class, but would it be worth adding tests for it?

Reply:
Yes! This totally slipped my mind.